NumLib/Extrapolation/LocalLinearLeastSquaresExtrapolator.h

```diff
@@ -82,17 +84,18 @@ class LocalLinearLeastSquaresExtrapolator : public Extrapolator
     void extrapolateElement(
         std::size_t const element_index, const unsigned num_components,
         ExtrapolatableElementCollection const& extrapolatables, const double t,
-        GlobalVector const& current_solution,
-        LocalToGlobalIndexMap const& dof_table, GlobalVector& counts);
+        std::vector<GlobalVector*> const& x,
+        std::vector<NumLib::LocalToGlobalIndexMap const*> const& dof_table,
+        GlobalVector& counts);
```

Comment:
I don't know how `counts` is used, but I wonder why isn't `counts` not a `std::vector<GlobalVector*>&`?

Reply:
If I remember correctly, the counts vector is for averaging the node values, so more topology related then DOF table. Anyway it's either correct or not tested. ;)